### DISCUSSION TITLE
I LOST MY CHIA'S FROM WALLET

### DISCUSSION THREAD

**Discussion by wrestler9595** [Chia-Network/chia-blockchain#10510] [Support]
Hello everyone, i don't have very good english so sorry for it.
I reinstalled my chia from 0 because the sync don't work, and when i connect to my wallet i see balance "0" i had 3.5... chia's before reconnection!

**Comment by loppefaaret** [comment]
i would suggest you try and upgrade to 1.3.0 release that has just been released. 
if the problem persists, please let me know
